js/views/modals/Settings/Moderation.js

```diff
@@ -21,52 +25,51 @@ export default class extends baseVw {
       this.profile.set('modInfo', this.moderator);
     }
 
+    this.currencyList = getTranslatedCurrencies(app.settings.get('language'));
+
     this.listenTo(this.profile, 'sync', () => {
       app.profile.set({
         moderator: this.profile.get('moderator'),
         modInfo: this.profile.get('modInfo').toJSON(),
       });
     });
-
-    // When this bug is fixed and we're saving via PATCH,
-    // the block below could be removed:
-    // https://github.com/OpenBazaar/openbazaar-go/issues/314
-    this.listenTo(app.profile, 'someChange', () => {
-      const profileAttrs = _.omit(app.profile.toJSON(),
-        ['moderator', 'modInfo']);
-      this.profile.set(profileAttrs);
-    });
   }
 
   events() {
     return {
-      'click .js-save': 'save',
+      'click .js-save': 'saveCheck',
+      'change #moderationFeeType': 'changeFeeType',
     };
   }
 
   getFormData() {
     return super.getFormData(this.$formFields);
   }
 
+  saveCheck() {
```

Comment:
Can we please integrate the save check into the save() method, for example?

```
    const formData = this.getFormData();
    this.profile.set(formData);

    if (this.profile.get('moderator') &&!this.$understandRequirements.prop('checked') && !this.$acceptGuidelines.prop('checked')) {
        this.$moderationConfirmError.toggleClass('hide', !blockSave);
        return;
    }
```

This would allow the save() to be self contained and, IMHO, make for a cleaner API. For example, let's say you wanted to initiate the saving of this form from a different view - all you'd need to do is call save(). With the current approach, you'd need to call saveCheck().